eng/CadlGenerate.ps1

```diff
@@ -1,5 +1,7 @@
 #Requires -Version 7.0
 param($filter)
+
+$ErrorActionPreference = 'Stop'
```

Comment:
It stops without error string. It is better to  check after command execution and give out error if fail.

Reply:
It will have error message. See example here: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1870922&view=logs&j=b5cdfb1e-c180-561f-918e-a169e451719e&t=aff748e3-476c-56d1-ae31-4029fde28653

Conversely, without this line the CI pipeline will always pass instead of stopping with error.